rustls/src/client/tls13.rs

```diff
@@ -390,7 +390,6 @@ fn validate_encrypted_extensions(
 pub struct ExpectEncryptedExtensions {
     pub handshake: HandshakeDetails,
     pub key_schedule: KeyScheduleHandshake,
-    pub server_cert: ServerCertDetails,
```

Comment:
`server_cert` is empty at this point so there's no point in threading it through yet.